tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java

```diff
@@ -2,14 +2,13 @@
 
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class TesAdapterController {
-
-    @GetMapping("/v1/tasks/service-info")
+    @RequestMapping(method = RequestMethod.GET, value = "/v1/tasks/service-info")
```

Comment:
why you decided to change GetMapping to RequestMapping
And also try to annotate this method with Swagger annotation to be able to provide information for swagger services

Reply:
@GetMapping is a composed annotation that acts as a shortcut for @RequestMapping(method = RequestMethod.GET).
So, they can easily replace each other